hols/oci-basic-setup/README.md

```diff
@@ -0,0 +1,203 @@
+    ```shell
+    cd ~/oci-mp
+    ```
+11. Run the `update_config_values.sh` utility script from the `oci-basic-setup` local repository to update the config parameters. Specify the directory location of the Helidon application as an argument to this script.
+    ```shell
+    ~/helidon-labs/hols/oci-basic-setup/update_config_values.sh ~/oci-mp
+    ```
+    Invoking this script performs the following:
+    1. Updates the `~/oci-mp/server/src/main/resources/application.yaml` config file to set up a Helidon feature that sends Helidon generated metrics to the OCI monitoring service.
+       1. compartmentId - The Compartment OCID used for this demo.
+       2. namespace - This can be any string but for this demo, will be set to `helidon_metrics`.
+    2. Updates the `~/oci-mp/server/src/main/resources/META-INF/microprofile-config.properties` config file to set up configuration parameters used by the Helidon app code to perform integration with OCI Logging and Metrics service.
+       1. oci.monitoring.compartmentId - Compartment ocid that is used for this demo
+       2. oci.monitoring.namespace - This can be any string but for this demo, this will be set to `helidon_application`.
+       3. oci.logging.id - Application log id that was provisioned by the terraform scripts.
+
+    **Note:** Make sure to validate that `application.yaml` and `microprofile-config.properties` were updated by checking that the mentioned config parameters were properly populated.
+12. To prepare for the build, ensure that JDK 21 and Maven 3.8+ exist and are set in the PATH environment variable of the Cloud Shell terminal.
+13. Build the application.  
+    ```shell
+    mvn clean package
+    ```
+
+### Deploy the Helidon OCI application:
+1. Go back to `oci-basic-setup` local repository directory.
+   ```shell
+   cd ~/helidon-labs/hols/oci-basic-setup
+   ```
+2. Open up `deploy.sh` script and use the JDK_TAR_GZ_INSTALLER variable to specify the download path of JDK 21 based on the OS flavor currently installed on Compute instance. The current value used is the Linux x64 version of JDK 21. 
+3. Run `deploy.sh` utility script and provide the directory of the OCI MP application as the argument. This script will zip up the server binaries, upload it to the Compute instance, install JDK 21, create a systemctl service for the application to ensure that the application is persistent across reboots, and start the service.
+   ```shell
+   ./deploy.sh ~/oci-mp
+   ```
+
+### Exercise the deployed Helidon application:
+1. Access the application by using curl to perform GET & PUT HTTP requests.
+    1. Set the endpoint using the instance's public ip which can be retrieved using `get.sh` utility script from the `oci-basic-setup` local repository directory.
+       ```shell
+       export ENDPOINT_IP=$(~/helidon-labs/hols/oci-basic-setup/get.sh public_ip)
+       echo "Instance public ip is $ENDPOINT_IP"
+       ```
+    2. Test the Hello world request.
```

Comment:
after this line maybe include: "These are some commands you can invoke on the Helidon application that you just deployed:"

Reply:
I think the step descriptuion indicating `Access the application by using curl to perform GET & PUT HTTP requests.` should be enough to convey the same message.